tests/test_api.py

```diff
@@ -1542,13 +1580,31 @@ def verify_build_json(passed_build_json):
         build_response = osbs_obj._create_scratch_build(build_request)
         assert build_response.json == {'spam': 'maps'}
 
-    @pytest.mark.parametrize(('nodeselector_str, nodeselector_dict'), [
-        (None, {}),
-        ('foo=  bar', {'foo': 'bar'}),
-        ('foo=bar ,  baz=foo', {'foo': 'bar', 'baz': 'foo'}),
+    @pytest.mark.parametrize(('platform, platform_node, nodeselector_str, nodeselector_dict'), [
```

Comment:
Better names for `platform_node` and `nodeselector_str` would now be `platform_nodeselector` and `lowpriority_nodeselector`.